### PR TITLE
[TT-1811] added documentation of slave options of gw

### PR DESCRIPTION
**Pull request by sredxny**
Added documentation for each of the slave options that we can set for a worker gateway in MDCB.

### CODE REVIEW

**Review by marksou — APPROVED** [review]

**Review by tbuchaillot — APPROVED** [review]
looks good!

**Review by matiasinsaurralde — APPROVED** [review]

File: tyk-docs/content/tyk-stack/tyk-gateway/configuration/tyk-gateway-configuration-options.md [hunk 1]
```diff
@@ -1001,9 +1001,50 @@ From v2.9.3 you can force the validation of the hostname against the common name
 From v3.0 you can log all the 404 errors happening if user tried to access Gateway with unknown listen path.
 The log level used for these records is Error and the feature can be enabled by setting the `config track_404_logs` to `true` in the gateway's config file.
 
-### key_space_sync_interval
+### Slave options
+In slave_options we will set the configurations that will enforce the RPC slave mechanism, this must be configured for any slave node. Next will be described each of the configurations available for slaved nodes
 
-From 3.0.1 you have the ability to set the interval's length in which the slaved gateway will check for changes in the key space, if this value is not set then by default it will be 10 seconds.
+#### slave_options.use_rpc
+Whether to connect to MDCB using rpc
```
Comment (gregdelhon) [inline tyk-docs/content/tyk-stack/tyk-gateway/configuration/tyk-gateway-configuration-options.md]: @marksou @sredxny as per conversation we need to change the wording for binary (true/false) options to something more explicit like 'Set to true to enable X"

Comment (marksou) [reply tyk-docs/content/tyk-stack/tyk-gateway/configuration/tyk-gateway-configuration-options.md]: @gregdelhon @sredxny Have made some changes

**Comment by gregdelhon** [comment]
@marksou ready to merge IMO

**Pull request merged**